package.json

```diff
@@ -13,7 +13,6 @@
   },
   "types": "dist/src/meriyah.d.ts",
   "typings": "dist/src/meriyah.d.ts",
-  "author": "KFlash",
```

Comment:
Removed author but didn't add to authors?

Reply:
> ci said package-lock is not in sync, pls do `npm i` and commit lock file.

I can't do it from a cp. Can you fix it?

> Removed author but didn't add to authors?

Didn't figure out how to write it in authors. It's only a nick that one. Can you take a stab at it?